### DISCUSSION TITLE
Does the "storage" folder need to be backed up or can it just be recreated?

### DISCUSSION THREAD

**Discussion by jftuga** [photoprism/photoprism#2437] [Q&A]
I am using `rclone` to backup my `Pictures` folder to BackBlaze.  I am also using `mysqldump` to backup the database.  Only after, I wrote these scripts did I learn about the built-in backup mechanism.  If I already have `Pictures` and the database backup file stored on Backblaze, would it still be necessary to backup the `storage` folder?  In a disaster-recovery scenario, could I restore the Pictures & Database and then through the UI, go to Library -> Import -> check `Complete Rescan`?  Will this repopulate the `storage` folder?

My end goal is to **not** backup the `storage` folder as it is 580,000 files.  I would prefer to rebuild it from the original photos & videos.

Thank you.

**Comment by krosseyed** [comment]
I am in a different scenario as I am able to backup a snapshot of my VM running PhotoPrism directly from my NAS, but it would be nice to know if I even need to worry about backing up the VM since its grabbing photos from the same storage device.

**Comment by dror3go** [comment]
https://docs.photoprism.app/getting-started/advanced/backups/
